sycl/test/on-device/online_compiler/online_compiler_OpenCL.cpp

```diff
@@ -0,0 +1,33 @@
+// REQUIRES: opencl
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -lOpenCL -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+#include <CL/sycl.hpp>
+#include <CL/sycl/INTEL/online_compiler.hpp>
+
+#include <vector>
+
+using byte = unsigned char;
+
+sycl::kernel getSYCLKernelWithIL(sycl::context &Context,
+                                 const std::vector<byte> &IL) {
+  cl_int Err;
+  cl_program ClProgram =
+      clCreateProgramWithIL(Context.get(), IL.data(), IL.size(), &Err);
+  if (Err != CL_SUCCESS)
+    throw sycl::compile_program_error();
```

Comment:
in the comment above you asked not using compile_program_error().
I will replace it with "throw sycl::runtime_error()" without arguments because we don't have other SYCL exceptions accepting only std::string.

Reply:
ok